install/playbooks/roles/system-prepare/defaults/main.yml

```diff
@@ -14,11 +14,12 @@ system_required:
   - nginx
   - acl
   - certbot
+  - python-netaddr
```

Comment:
I don't think this is needed for the target server.
I wasn't specific enough in my comment: the dependency I was talking about is on the host running the ansible playbooks.
As it concerns only the ansible host, I thought an addition to the docs would be sufficient. Currently, in `docs/prerequisites.md` there is an example that will still be valid (as ansible depends on python3-netaddr on debian and ubuntu). But for other platforms, the installation of the python `netaddr` module might be required.